lib/iden3comm/data/data_sources/storage_connection_data_source.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:injectable/injectable.dart';
+
+import 'package:polygonid_flutter_sdk/common/utils/encrypt_sembast_codec.dart';
+import 'package:polygonid_flutter_sdk/constants.dart';
+import 'package:polygonid_flutter_sdk/iden3comm/data/dtos/connection_dto.dart';
+import 'package:polygonid_flutter_sdk/sdk/di/injector.dart';
+import 'package:sembast/sembast.dart';
+import 'package:sembast/utils/sembast_import_export.dart';
+
+/// [StoreRef] wrapper
+/// Delegates all call to [ConnectionStoreRefWrapper._store]
+/// Needed for UT for mocking extension methods
+@injectable
+class ConnectionStoreRefWrapper {
+  final Map<String, StoreRef<String, Map<String, Object?>>> _store;
```

Comment:
Only inject the source of this DataSource, not all Stores (see my comment below).

Reply:
will do in another PR